src/workers/userCreatedAddPersonalizedDigest.ts

```diff
@@ -0,0 +1,22 @@
+import { User } from '../common';
+import { UserPersonalizedDigest } from '../entity/UserPersonalizedDigest';
+import { messageToJson, Worker } from './worker';
+
+interface Data {
+  user: User;
+}
+
+const worker: Worker = {
```

Comment:
Can be substituted with a trigger, can't it?

Reply:
Conditions on which we enroll might change based on multiple discussion so I think having a worker gives a bigger flexibility. If logic in triggers can get ugly.